plugins/datadog/app/models/datadog_monitor.rb

```diff
@@ -5,18 +5,20 @@ class DatadogMonitor
   API_KEY = ENV["DATADOG_API_KEY"]
   APP_KEY = ENV["DATADOG_APPLICATION_KEY"]
   SUBDOMAIN = ENV["DATADOG_SUBDOMAIN"] || "app"
+  BASE_URL = "https://#{SUBDOMAIN}.datadoghq.com"
```

Comment:
```suggestion
  BASE_URL = ENV["DATADOG_URL"]
```
since datadoghq.eu is also possible

Reply:
oh boy

Reply:
added